cypress/support/pages/home.pageObject.js

```diff
@@ -3,9 +3,21 @@ import PageObject from '../PageObject';
 class HomePageObject extends PageObject {
   url = '/#/';
 
+  visitYourFeed() {
```

Comment:
The method visitYourFeed() is not following the PageObject pattern. It contains navigation logic which should be avoided. The PageObject should return elements, not perform actions.